content/sensu-go/5.12/reference/agent.md

```diff
@@ -1202,6 +1203,41 @@ required              | false
 type                  | Boolean
 example               | {{< highlight shell >}}"enable_env": true{{< /highlight >}}
 
+### Configuring Flags as Environment Variables
+
+The `sensu-agent` service can read environment variables from `/etc/default/sensu-agent` on Debian/Ubuntu systems and `/etc/sysconfig/sensu-agent` on RHEL systems. These files are not created by the installation package and will need to be created.
+
+{{< language-toggle >}}
+
+{{< highlight "Ubuntu/Debian" >}}
+$ sudo touch /etc/default/sensu-agent
+{{< /highlight >}}
+
+{{< highlight "RHEL/CentOS" >}}
+$ sudo touch /etc/sysconfig/sensu-agent
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+For any flag you are configuring as an environment variable, you will need to append `sensu_` and convert dashes (`-`) to underscores (`_`). This will then need to be added to the appropriate file created above and the service will need to be restarted.
+
+In the following example `api-host` flag is configured as an environment variable and set to `"0.0.0.0"`.
+
+{{< language-toggle >}}
+
+{{< highlight "Ubuntu/Debian" >}}
+$ echo 'sensu_api_host="0.0.0.0' | sudo tee /etc/default/sensu-agent
```

Comment:
use `tee -a` so that we append to instead of overwrite the existing file

```suggestion
$ echo 'sensu_api_host="0.0.0.0' | sudo tee -a /etc/default/sensu-agent
```